pubtools/_quay/utils/misc.py

```diff
@@ -200,14 +200,22 @@ def get_internal_container_repo_name(external_name):
     Expected input format: <namespace>/<product>
     Generated output format: <namespace>----<product>
 
+    NOTE: Repositories without a delimeter "/" may actually exist. In that case, the function
+    simply returns the repo without any alterations.
+
     Args:
         external_name (str):
             External repository name.
     Returns:
         Internal repository name.
     """
-    if external_name.count("/") != 1 or external_name[0] == "/" or external_name[-1] == "/":
-        raise ValueError("Input repository should have the format '<namespace>/<product>'")
+    if external_name.count("/") == 0:
+        return external_name
+
+    if external_name.count("/") > 1 or external_name[0] == "/" or external_name[-1] == "/":
+        raise ValueError(
```

Comment:
It would be great if you could display the input repository in the error like you have done in line 250

Reply:
Added.